src/main.rs

```diff
@@ -73,12 +75,20 @@ async fn main() -> bluer::Result<()> {
     );
     let mut manufacturer_data = BTreeMap::new();
     manufacturer_data.insert(MANUFACTURER_ID, MANUFACTURER_ID_VAL.to_vec());
+    let local_name = match env::var("SCAN_SERVICE_BEACON") {
```

Comment:
SERVICE_BEACON vs SCAN_SERVICE_BEACON
vs
DEFAULT_SERVICE_BEACON vs DEFAULT_SCAN_SERVICE_BEACON

Reply:
changed to [DEFAULT_]SCAN_SERVICE_BEACON